plugin/handlergen.go

```diff
@@ -161,9 +161,9 @@ func (p *OrmPlugin) generateListHandler(message *generator.Descriptor) {
 
 	p.P(`// DefaultList`, typeName, ` executes a gorm list call`)
 	p.P(`func DefaultList`, typeName, `(ctx context.Context, db *`, p.Import(gormImport), ``,
-		`.DB) ([]*`, typeName, `, error) {`)
+		`.DB, params interface{}) ([]*`, typeName, `, error) {`)
 	p.P(`ormResponse := []`, ormable.Name, `{}`)
-	p.P(`db, err := `, p.Import(tkgormImport), `.ApplyCollectionOperators(db, ctx)`)
+	p.P(`db, err := `, p.Import(tkgormImport), `.ApplyCollectionOperators(db, params)`)
```

Comment:
The function signature is changing in atlas-app-toolkit? 
And this requires that any/all operators be included in the request, I guess this makes sense, but has there been a consensus on this?

Reply:
I also send for review changes in atlas-app-toolkit.